cfgov/v1/jinja2/v1/layouts/_last-modified.html

```diff
@@ -0,0 +1,6 @@
+{% if page.last_published_at %}
+<div class="block">
+    <p><strong>Page last updated {{ page.last_published_at.strftime('%Y-%m-%d, %I:%M:%S %p %z') }}.</strong></p>
```

Comment:
Could you use one of our existing macros for this, for example [`format_date`](https://github.com/cfpb/consumerfinance.gov/blob/c1d827ae170079c401a6ca71c992eb79e37125e8/cfgov/v1/jinja2/v1/includes/macros/util/format/datetime.html#L48)?

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/8778/commits/5d4b829b542589d0a687e2153db523e21a0a1244